frontend/src/components/CardBoard/CardAvatars.tsx

```diff
@@ -21,10 +22,26 @@ type CardAvatarProps = {
 	userId: string;
 	myBoards?: boolean;
 	haveError?: boolean;
+	isBoardsPage?: boolean;
 };
 
 const CardAvatars = React.memo<CardAvatarProps>(
-	({ listUsers, teamAdmins, stakeholders, userId, haveError, responsible, myBoards }) => {
+	({
+		listUsers,
+		teamAdmins,
+		stakeholders,
+		userId,
+		haveError,
+		responsible,
+		myBoards,
+		isBoardsPage
+	}) => {
+		const [viewAllUsers, setViewAllUsers] = useState(false);
+
+		const handleViewAllUsers = useCallback(() => {
+			setViewAllUsers(!viewAllUsers);
+		}, [viewAllUsers]);
```

Comment:
```suggestion
		const handleViewAllUsers = () => {
			setViewAllUsers(!viewAllUsers);
		};
```

Do you really need this useCallback?

Reply:
In the start I had as you suggested but eslint advised me to useCallback because it makes the dependencies of renderAvatar change on every render. But If you think that there is no problem with using your approach I can put an 	`// eslint-disable-next-line react-hooks/exhaustive-deps ` before the function and remove the useCallback

Reply:
No problem, we can continue with your approach